### PR TITLE
{lib}[foss/2022a] scikit-lego v0.6.16 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node301.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/f2cbc6a87b6683c599c79b33d2422a85 for a full test report.

File: easybuild/easyconfigs/s/scikit-lego/scikit-lego-0.6.16-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    ('autograd', '1.6.2', {
+        'checksums': ['8731e08a0c4e389d8695a40072ada4512641c113b6cace8f4cfbe8eb7e9aedeb'],
+    }),
+    ('Deprecated', '1.2.14', {
+        'checksums': ['e5323eb936458dccc2582dc6f9c322c852a775a27065ff2b0c4970b9d53d01b3'],
+    }),
+    ('patsy', '0.5.3', {
+        'checksums': ['bdc18001875e319bc91c812c1eb6a10be4bb13cb81eb763f466179dca3b67277'],
+    }),
+    ('wrapt', '1.14.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-lego/scikit-lego-0.6.16-foss-2022a.eb:41]: @WilleBell We have an easyconfig for `wrapt` 1.15.0 with `foss/2022a`, any reason you're not using that as a dependency?

File: easybuild/easyconfigs/s/scikit-lego/scikit-lego-0.6.16-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+models. The goal of this project is to attempt to consolidate these into a package that offers
+code quality/testing. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'koaning'
+
+builddependencies = [
+    ('binutils', '2.38'),
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-lego/scikit-lego-0.6.16-foss-2022a.eb:18]: `binutils` is not required when using `foss` toolchain (only when using `GCCcore` as toolchain)

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3153.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/7d48ab06fcaeeb8990123fa12c3b67f1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @WilleBell!

**Pull request merged**